mmeds/util.py

```diff
@@ -182,6 +182,58 @@ def join_metadata(subject, specimen, subject_type):
     return df
 
 
+def split_metadata(full, clean_for_meta_analysis=True, id_col=('RawData', 'RawDataID'), new_study_name=None):
+    """ Splits a full metadata file into two dataframes, subject and specimen """
+    # Determine subject column set and subject ID col
+    tables = [header[0] for header in full.columns]
+    if 'Ethnicity' in tables and 'Chow' in tables:
```

Comment:
weak if statements, the kind of thing that can break if column names are ever changed. See if there's a better way